datafusion/sql/src/unparser/expr.rs

```diff
@@ -561,6 +636,31 @@ mod tests {
                 }),
                 r#"CAST("a" AS INTEGER UNSIGNED)"#,
             ),
+            (
+                Expr::InList(InList {
+                    expr: Box::new(col("a")),
+                    list: vec![lit(1), lit(2), lit(3)],
+                    negated: false,
+                }),
+                r#""a" IN (1, 2, 3)"#,
+            ),
+            (
+                Expr::InList(InList {
+                    expr: Box::new(col("a")),
+                    list: vec![lit(1), lit(2), lit(3)],
+                    negated: true,
+                }),
+                r#""a" NOT IN (1, 2, 3)"#,
+            ),
+            (
+                Expr::ScalarFunction(ScalarFunction {
+                    func_def: ScalarFunctionDefinition::UDF(Arc::new(
+                        ScalarUDF::new_from_impl(DummyUDF::new()),
+                    )),
+                    args: vec![col("a"), col("b")],
+                }),
```

Comment:
How about using the call API https://docs.rs/datafusion/latest/datafusion/logical_expr/struct.ScalarUDF.html#method.call

```rust
ScalarUDF::new_from_impl(DummyUDF::new())
 .call(vec![col("a"), col("b")])
```

Reply:
Love this, more concise!